browser/js/content.js

```diff
@@ -5,10 +5,51 @@
 const asset = chrome.runtime.getURL;
 
 function init() {
-  // Only do work on reddit.
-  if (!location.hostname.endsWith('.reddit.com')) return;
+  // Passive queries?
+  chrome.storage.sync.get("profile-passive-queries", function(options) {
+    if (options["profile-passive-queries"] !== true) return; // Default false
+
+    if (location.hostname.endsWith('twitter.com')) {
```

Comment:
could someone register a domain "blahtwitter.com" and do something bad?Maybe should be '.twitter.com'?